recipes/glog/all/conanfile.py

```diff
@@ -56,7 +71,18 @@ def requirements(self):
             self.requires("gflags/2.2.2", transitive_headers=True, transitive_libs=True)
         # 0.4.0 requires libunwind unconditionally
         if self.options.get_safe("with_unwind") or (Version(self.version) < "0.5.0" and self.settings.os in ["Linux", "FreeBSD"]):
-            self.requires("libunwind/1.7.2")
+            self.requires("libunwind/1.8.0", transitive_headers=True, transitive_libs=True)
+
+    def validate(self):
+        if Version(self.version) < "0.7.0":
+            return
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
 
     def build_requirements(self):
         if Version(self.version) >= "0.6.0":
```

Comment:
```suggestion
        if Version(self.version) >= "0.6.0":
```

The minimal CMake version in 0.7 is 3.22, but the current one listed is 3.16

Reply:
@uilianries 
Thanks a lot!
Fixed.